values.yaml

```diff
@@ -71,10 +71,9 @@ devurls:
 # connections to environments such as IDE or shell sessions.
 envproxy:
   # envproxy.accessURL -- The URL reported to cemanager. Must be accessible by
-  # cemanager and all users who can use this workspace provider. Derived from
-  # ingress.host if not set. This should be a full URL, complete with protocol
-  # and trailing "/proxy" (no trailing slash).
```

Comment:
nitpick: maybe we use `wsp.coder.com` or `example.coder.com` to keep it generic, since a lot of the wsp docs don't reference the proxy directly.